### ISSUE TITLE
SlevomatCodingStandard.ControlStructures.NewWithParentheses.MissingParentheses false positive

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1288]
PHP 8.0 + master (c7d4801)

```php
<?php declare(strict_types = 1);

$c = new (stdClass::class)();

echo $c::class;
```

```
FILE: /project/packages/Bug.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 3 | ERROR | [x] Usage of "new" without parentheses is disallowed.
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 992ms; Memory: 14MB
```
